ISSUE_TEMPLATE/new_campaign.md

```diff
@@ -2,6 +2,8 @@ ___ DELETE ___
 
 [click here to create a new campaign tracking_issue](https://github.com/rust-community/content-o-tron/issues/new?template=new_campaign.md&title=Tracking:+__NAME_OF_CAMPAIGN__&labels=campaign)
 
+[Have a look at the Running a blog post campaign](https://github.com/rust-community/content-o-tron/blob/master/guides/running_a_campaign.md)
```

Comment:
It might be better to have this link before the "click here to create a new...", the idea of the new link is to give you guidance *before* you create a new campaign.

Also the wording for the new link could be more meaningful, "Are you new to running a blogging campaign? Then have a look at our handy [guide] to learn more."